packages/medusa/src/services/__tests__/claim.js

```diff
@@ -208,6 +211,25 @@ describe("ClaimService", () => {
         })
       ).rejects.toThrow(`Claims must have at least one claim item.`)
     })
+
+    it.each(
+      [
+        [false, false],
+        [undefined, true],
+      ],
+      "passes correct no_notification status to event bus",
+      async (input, expected) => {
+        await claimService.create({
+          ...testClaim,
+          no_notification: input,
+        })
+
+        expect(eventBusService.emit).toHaveBeenCalledWith(expect.any(String), {
+          id: expect.any(String),
+          no_notification: expected,
+        })
+      }
+    )
```

Comment:
Default setup for testing the logic added; If passed a flag, expect it to be this, otherwise inherit from object above (i.e. order / claim / fulfillment / swap) and expect that value (in all tests these are set to true)